### PR TITLE
[docs] Changed manual on how to run docs locally

### PR DESCRIPTION
**Pull request by mortelumina98**
## Description

   Changed manual on how to install the documentation locally.

## Why do we need it, and what problem does it solve?

  Solves the problem of incorrect installation of documentation via the werf compose up method in /docs/site .

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: chore
summary: Changed the manual on how to install the documentation locally.
impact_level: low
```

### CODE REVIEW

File: docs/site/LOCAL_DEV.md [hunk 1]
```diff
@@ -1,61 +1,66 @@
...
+- Clone repo firstly.
 
 - Free 80 port to bind.
 
-- Install werf
+- Install werf.
 
-- Open console and start documentation container with one of the following methods:
-  - Using makefile:
+- Open console and running documentation and site containers with one of the following methods:
```
Comment (z9r5) [inline docs/site/LOCAL_DEV.md:9]:
```suggestion
- Open console and run documentation and site containers with one of the following methods:
```

File: docs/site/LOCAL_DEV.md [hunk 2]
```diff
@@ -1,61 +1,66 @@
...
 
+  - The first method — open two consoles and run containers using the Makefile:
```
Comment (z9r5) [inline docs/site/LOCAL_DEV.md:11]:
```suggestion
  - **The first method** — open two consoles and do the following steps.
```

File: docs/site/LOCAL_DEV.md [hunk 3]
```diff
@@ -1,61 +1,66 @@
...
+
+    1. In the first console run:
+		
     ```shell
     cd docs/documentation
     make up 
     ```
```
Comment (z9r5) [inline docs/site/LOCAL_DEV.md:18]: @mortelumina98 Please add indent with three spaces to the code block here and in the similar cases below

File: docs/site/LOCAL_DEV.md [hunk 4]
```diff
@@ -1,61 +1,66 @@
...
-    make up 
+    make dev
     ```
-
-    > For development mode use `make dev` instead.
-
-  - or using the following commands:
-
+	
+  - The second method — open console and run containers in a single iteration using Makefile in the root of the repo:
```
Comment (z9r5) [inline docs/site/LOCAL_DEV.md:43]:
```suggestion
  - **The second method** — open console and run the necessary containers in the root of the repo:
```

File: docs/site/LOCAL_DEV.md [hunk 5]
```diff
@@ -1,61 +1,66 @@
...
+    For development mode use `make docs-dev` instead:
+	
+	  ```shell
+    cd deckhouse
+    make docs-dev 
+    ```
+	
 - Open <http://localhost>.
 
 Don't forget to stop documentation and site containers by running:
```
Comment (z9r5) [inline docs/site/LOCAL_DEV.md:59]:
```suggestion
- To stop the running containers use the following command:
  - For the first method (in the each console):
    ```shell
    make down
    ```
  - For the second method:
    ```shell
    make docs-down
    ```
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

**Pull request merged**